agent/consul/leader.go

```diff
@@ -979,7 +979,7 @@ func (s *Server) startConfigReplication() {
 		return
 	}
 
-	s.leaderRoutineManager.Start(configReplicationRoutineName, s.configReplicator.Run)
+	s.leaderRoutineManager.Start(context.Background(), configReplicationRoutineName, s.configReplicator.Run)
```

Comment:
How come these don't use the context from parent functions like `establishLeadership`? Should these be `context.TODO()`?

Reply:
I didn't look far enou

> Did you consider other names for the gort package? This abbreviation seems a bit unintuitive. What if it was called `routine`?

I started from goroutine and just shortened it. I would be happy to rename it to `routine`.